BenchmarkVCFs/BenchmarkVCFs.wdl

```diff
@@ -163,7 +166,8 @@ workflow Benchmark {
                         refDict=refDict,
                         gatkTag=gatkTag,
                         subset_interval=CreateIntervalList.interval_list,
-                        preemptible=preemptible
+                        preemptible=preemptible,
+                        dummyInputForTerraCallCaching = dummyInputForTerraCallCaching
```

Comment:
Stylistic comment: the spaces around `=` should be removed here, so that it says`dummyInputForTerraCallCaching=dummyInputForTerraCallCaching` to be consistent with previous lines. (Of course, this doesn't seem to be a consistent convention throughout the file but at least is held locally…)